content/academic_update/waiting/_index.md

```diff
@@ -0,0 +1,74 @@
+{{< figure src="waitingtime.png" caption="Waiting places depending on total waiting time a) up to 2 min b) 2-5 min c) 5-10 min d) 10 min and more" >}}
+
+
+## Experiments
+As the field studies revealed a higher number of influencing factors, in the next step, large scale laboratory experiments were conducted [[3]](#Küpper3). These offer the possibility to study some influencing factors in a controlled environment and under reduced complexity. 
+As variations the setup of the platform (no, narrow or wide obstacle), the number of participants (40 or 100) and their waiting time (2 or 4 minutes) were chosen. The side of train arrival  was not varied and known to the participants. Hence, their attention is focused towards the corresponding side and an alignment of viewing directions can be observed.
+
+{{< figure src="exp_density.png" caption=" Upper panel: experiment setup and participant's positions at the end of the waiting time; middle: density profiles for 2 minutes waiting time and 40 Participants; lower panel: for 100 participants" >}}
+
+During analysis of density profiles, platforms without obstacles were found to lead to uneven passenger distributions as the area close to the entrance was favoured. Obstacles were found to have a two-sided effect: the side facing towards the train arrival is a preferred waiting spot while the opposite side has a repulsive effect. Possible reasons for these effects are the comfort provided by leaning against the obstacle and the reduction of the number of neighbouring persons; the repulsive effects are likely caused by the blocked line of sight towards the goal. Narrow obstacles cause passengers to mainly wait at the side of expected train arrival, while wide obstacles, despite blocking the passenger's view, lead to a more even distribution along the platform.  Even though the marked safety line at the platform's edges was not mentioned in the experiment's instructions, participants keep a distance to the edges and seldom cross the safety line.
+
+### Floor fields
+Based on the results from both field and experimental studies, pedestrians choose their waiting positions as a trade-off between different factors. These factors can be transferred into floor fields, as previously introduced by [[5]](#Ezaki).
+{{< figure src="floorfields.png" caption="Schematic illustration of influencing factors" >}}
+
+a) Distance to entrance: Passengers prefer places in the vicinity of the entrance and do not walk to the far side of the platform.
+
+b) Distance to exit: Passengers stay at the side of expected train arrival.
+
+c) Hazard zones: Platform edges have a repulsive effect. 
+
+d) Flow avoidance: Passengers avoid places where they often get in the way of others and are perceived as obstacle.
+
+e) Stationary obstacles: Obstacles have a two-sided effect: the side facing train arrival acts attractive, the opposite side as repulsive.
```

Comment:
Please change style to list

Reply:
added bullet points. markdown refused to add a lettered list...